plugin/src/main/kotlin/com/automattic/android/CheckS3VersionTask.kt

```diff
@@ -10,21 +10,23 @@ import org.gradle.api.tasks.Internal
 
 abstract class CheckS3VersionTask : DefaultTask() {
     @Internal
-    override fun getDescription(): String = "Checks if a version is already published to S3"
+    override fun getDescription(): String = "Checks if a version is published to S3"
 
     @get:Input
+    @set:Option(option = "published-group-id", description = "Published group id")
```

Comment:
Nitpick

```suggestion
    @set:Option(option = "published-group-id", description = "Published artifact group id")
```

I'm suggesting this because my understanding is that we're looking for _the group id of the published artifact_. Reading "published group id" makes me think we're looking for _the id of a published group_.